etc/kernel-launchers/scala/toree-launcher/build.sbt

```diff
@@ -6,15 +6,14 @@
 name := "toree-launcher"
 
 version := sys.props.getOrElse("version", default = "1.0").replaceAll("dev[0-9]", "SNAPSHOT")
+val sparkVersion = sys.props.getOrElse("spark_version", default = "2.4.6")
```

Comment:
Does this buy us anything if we always have to update the `default` anyway?  If this works, and the default is `2.4.6` because its the first use - that's cool - I'm good with this.